test/swagger/alias.json

```diff
@@ -392,6 +392,7 @@
       }
     },
     "GeoJsonObjectNamedCollection": {
+      "x-ms-external": true,
```

Comment:
I suppose you would do this instead of simply deleting the definition when the definition is referenced by something you do want to generate?

Reply:
Correct.  This way, you can have a hand-written model used by generated code.